cfme/tests/infrastructure/test_rest_providers.py

```diff
@@ -0,0 +1,126 @@
+import fauxfactory
+import pytest
+
+from cfme.configure.configuration import server_roles_disabled
+from cfme.rest import a_provider as _a_provider
+from utils.virtual_machines import deploy_template
+from utils.version import current_version
+from utils.wait import wait_for
+from utils import testgen
+
+pytest_generate_tests = testgen.generate(
```

Comment:
Just a quick question, I don't see the fixture `provider` used by any of the tests, is the parametrization here really necessary? Since you use `a_provider`.